aspnetcore/mvc/views/tag-helpers/intro.md

```diff
@@ -58,7 +58,7 @@ If you create a new ASP.NET Core web app named *AuthoringTagHelpers* (with no au
 
 The `@addTagHelper` directive makes Tag Helpers available to the view. In this case, the view file is *Views/_ViewImports.cshtml*, which by default is inherited by all view files in the *Views* folder and sub-directories; making Tag Helpers available. The code above uses the wildcard syntax ("*") to specify that all Tag Helpers in the specified assembly (*Microsoft.AspNetCore.Mvc.TagHelpers*) will be available to every view file in the *Views* directory or sub-directory. The first parameter after `@addTagHelper` specifies the Tag Helpers to load (we are using "*" for all Tag Helpers), and the second parameter "Microsoft.AspNetCore.Mvc.TagHelpers" specifies the assembly containing the Tag Helpers. *Microsoft.AspNetCore.Mvc.TagHelpers* is the assembly for the built-in ASP.NET Core Tag Helpers.
 
-To expose all of the Tag Helpers in this project (which creates an assembly named *AuthoringTagHelpers*), you would use the following:
+It is important to understand that if you author your Tag Helpers in the same project as your MVC web application (this project), that you must explicitly include the assembly name of this project (which is *AuthoringTagHelpers*).  If you do not include your current executing assembly in your *Views/_ViewImports.cshtml* file, your authored Tag Helpers will not be available in your view cshtml files.  Specifically, you would use the following:
```

Comment:
@Rick-Anderson somehow, I'm getting confused between pull requests.  The diff's above show my first PR that had to do with _ViewImports.cshtml.  I'm good with closing and abandoning that one.  But, the title on top of my web page says "Update File Locations and SpeakerController.cs" which I issues yesterday and that had small changes to the previous pull request that was all about tag helpers.